src/integrationTest/java/uk/gov/hmcts/reform/userprofileapi/client/UserProfileRequestHandlerTest.java

```diff
@@ -146,7 +143,7 @@ public MvcResult sendPut(MockMvc mockMvc,
                 .headers(getMultipleAuthHeaders())
                 .content(jsonBody)
                 .contentType(APPLICATION_JSON))
-                .andExpect(status().is(expectedHttpStatus.value())).andReturn();
```

Comment:
any reason for removing this? Can you point the code where you are really asserting the http status value?